checks.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env bash
+
+for i in *; do
+    if [[ -d $i ]]; then
+        nix develop $i/ -c echo "test" || exit 1
```

Comment:
Can you change this so that it continues testing all templates even if one fails? Then at the end, if any single template failed, exit with status 1.

Reply:
Also keep track of which templates failed.

Reply:
done

Reply:
Print which templates failed too, so we dont have to pore through the logs when there are a bunch of templates

Reply:
*now* it's done